Tests/LibWeb/Text/input/wpt-import/css/css-ui/caret-color-009.html

```
@@ -0,0 +1,49 @@
+  function(){
+   var d2 = document.getElementById("d2");
+   assert_equals(window.getComputedStyle(d2)["caret-color"], "rgb(0, 255, 255)");
+  }, "Check the resolved value of 'currentcolor'");
+test(
+  function(){
+   var d3 = document.getElementById("d3");
+   assert_equals(window.getComputedStyle(d3)["caret-color"], "rgb(255, 0, 255)");
+  }, "Check the resolved value of 'initial'");
+</script>
+</body>
+</html>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </html> ]

(tag-pair)